cluster/src/main/java/com/hartwig/pipeline/CommandLineOptions.java

```diff
@@ -59,7 +63,7 @@
     private static final String SET_ID_FLAG = "set_id";
     private static final String SBP_RUN_ID_FLAG = "sbp_run_id";
     private static final String NETWORK_FLAG = "network";
-    private static final String CMEK_FLAG = "cmek";
+//    private static final String CMEK_FLAG = "cmek";
```

Comment:
I think we should keep all flags used in the CLI in this class for now or do a larger refactor. Prefer the consistency to the small duplication between CommonArguments and this list.